index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Alzubair Protfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/de9dd03ce2.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header class="container">
+    <h1 class="navbar">My Logo</h1>
+    <i class="fa-solid fa-bars"></i>
+  </header>
+
+  <div class="body">
+    <img src="Microverse-Student-Project-1/img.png" alt="background-img">
+    <div class="paragraph">
+      <h1 class="first-text">I’m Alzubair Glad to see you!</h1>
+      <h3>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h3>
```

Comment:
- [ ] please try to follow the figma characteristics of exact margins, font weight, line height, sizes, family, etc so it looks similar. the social media icons should have the same [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) color as well.
- your design 👇🏾 
![Screenshot (54)_LI](https://user-images.githubusercontent.com/65298048/170442471-ceaf17aa-f6cb-428b-a7e2-ee5b5c2226f5.jpg)
- figma design 👇🏾 
![Screenshot (53)_LI](https://user-images.githubusercontent.com/65298048/170442533-37d6ec6f-5510-43d7-839a-bac922bb3b47.jpg)

Reply:
hello dear 
I have fixed every thing you asked for , I did my best 
can you check it now?